scripts/installUbuntuDeps.sh

```diff
@@ -17,10 +17,10 @@ FAST_MAKE=''
 gcc_version=0
 
 check_version(){
-  if [[ $(lsb_release -rs) == "18.04" ]] || [[ $(lsb_release -rs) == "20.04" ]] 
-  then 
+  if [[ $(lsb_release -rs) == "18.04" ]] || [[ $(lsb_release -rs) == "20.04" ]]
```

Comment:
Do we still need to keep this? I'm asking because I'm not sure this will even work with older gcc versions